sdcm/cluster.py

```diff
@@ -2967,7 +2967,7 @@ def __init__(self, targets, params):
         self.sct_ip_port = self.set_local_sct_ip()
         self.grafana_port = 3000
         self.monitor_branch = self.params.get('monitor_branch', default='branch-2.3')
-        self.monitor_install_path_base = "/home/{}".format(self.params.get('ami_monitor_user'), default='centos')
+        self.monitor_install_path_base = "~"
```

Comment:
this cause exception if files were copied via scp
Command: 'scp -r -o StrictHostKeyChecking=no -o BatchMode=yes -o ConnectTimeout=300 -o ServerAliveInterval=300 -o UserKnownHostsFile=/var/tmp/tmpEwMudZ -P 22 -i /home/abykov/.ssh/scylla-qa-ec2 /var/tmp/tmpFhuyC4/prometheus.yml.template \'centos@3.93.183.21:"~/scylla-grafana-monitoring-branch-2.3/prometheus/prometheus.yml.template"\''

Reply:
if 'ami_monitor_user' in self.params:
            self.monitor_install_path_base = "/home/{}".format(self.params.get('ami_monitor_user'), default='centos')
        else:
            self.monitor_install_path_base = "~"

possible fix

Reply:
I think more robust way is to get the current user HOME directory and store it as a class attribute, then use it

Reply:
echo $HOME returns you the home directory for the current user